### PR TITLE
Added minimum password length requirement

### LINKED ISSUE #20: Add minimum password length requirements
We'd talked about enforcing minimum password length policies, so people don't use overly simple passwords. (I think there's some validation in the same code, but it is pretty basic: just "is the password non-empty.")

### PR DESCRIPTION
**Pull request by Mariamcm90**
Added a section to verify password at registration is at least 8 characters as well as a test for this change.
Fixes #20

### CODE REVIEW

**Comment by Mariamcm90** [comment]
Sure! I'll add that in.

File: src/main/java/codeu/controller/RegisterServlet.java [hunk 1]
```diff
@@ -54,6 +54,12 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
       return;
     }
 
+    if (password != null && password.length() < 8) {
```
Comment (fatlotus) [inline src/main/java/codeu/controller/RegisterServlet.java]: Maybe do `password == null ||`? That way we catch the no password case.

**Comment by Mariamcm90** [comment]
Fixed it, thanks!

File: src/test/java/codeu/controller/RegisterServletTest.java [hunk 2]
```diff
@@ -90,6 +92,18 @@ public void testDoPost_ExistingUser() throws IOException, ServletException {
     Mockito.verify(mockUserStore, Mockito.never()).addUser(Mockito.any(User.class));
   }
 
+  @Test
+  public void testDoPost_PasswordUnderMinLength() throws IOException, ServletException {
```
Comment (fatlotus) [inline src/test/java/codeu/controller/RegisterServletTest.java]: Maybe also add a case for no password entered?

Comment (Mariamcm90) [reply src/test/java/codeu/controller/RegisterServletTest.java]: Sure thing! Added it.

**Pull request merged**